blocks/library/paragraph/index.js

```diff
@@ -194,6 +167,86 @@ registerBlockType( 'core/paragraph', {
 				) }
 			</Autocomplete>,
 		];
+	}
+}
+
+const ParagraphBlockFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, backgroundColor } = ownProps.attributes;
+	const computedStyles = ( ! textColor || ! backgroundColor ) ? getComputedStyle( node.querySelector( '[contenteditable="true"]' ) ) : null;
+	return {
+		fallbackBackgroundColor: backgroundColor ? undefined : computedStyles.backgroundColor,
```

Comment:
Can you explain why we set `undefined` when color is passed as a prop? From what I can see we can safely set it, but it will be ignored. It would make reading code easier if we would always compute it even when it's not used. Unless I missed something.

Reply:
If there are custom colors set the fallbackColors will be equal to the custom colors set. As we have a logic that stops grabbing colors when we have all fallbackColors needed, if later we remove the custom colors, the fallbackColors are not updated and would still be equal to the custom colors set the first time instead of the colors that appear when no custom color is selected.